java/arcs/core/policy/PolicyOptions.kt

```diff
@@ -0,0 +1,14 @@
+package arcs.core.policy
+
+import arcs.core.data.StoreId
+
+/** Config options for policy enforcement. */
+data class PolicyOptions(
+    /**
+     * Maps from store ID to the schema name of the type it stores. Indicates all of the stores
+     * protected by policy
+     *
+     * Temporary measure until we have a proper way to designate these stores.
+     */
+    val storeMap: Map<StoreId, String>
```

Comment:
We will also need a map from `EgressType` to particle class? Is the plan to do it in a separate PR?

Reply:
I wasn't sure exactly what the mapping there was (1:1? 1:N?) so I thought I would leave it until we needed it and knew exactly what it was.